services/crypto/key/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+title: Key Management
+id: CCC.Key
```

Comment:
Should we change to something like "CCC.KeyMgmt"? Could jut be me, but just using "Key" doesn't seem very clear to me (same with the containing folder). But I am not a regular visitor to this space, so it may make more sense to those who are more familiar with this area?

Reply:
I agree with Stevie here. Making it clear we're talking about Key Management would be useful.